web-extensions/chrome/engine/messaging.js

```diff
@@ -0,0 +1,59 @@
+
```

Comment:
Want to add headers later on? (since you've left empty space...)

Reply:
Not voluntary but makes sense for licensing